llama_index/readers/weaviate/client.py

```diff
@@ -174,51 +185,29 @@ def _to_node(entry: Dict) -> Node:
 
 def _node_to_dict(node: Node) -> dict:
```

Comment:
do we need this function? doesn't seem used

Reply:
removed